doc/examples/read_experiment_and_data/example_experiment_data.py

```diff
@@ -52,7 +54,7 @@ def __init__(self):
         )
 
     def run(self):
-        from dials.array_family import flex  # import dependency
+        from dials.array_family import flex  # noqa F401 import dependency
```

Comment:
Is this _really_ an import dependency? If so, it ideally _shouldn't_ be, I'd have thought?

Reply:
I agree - I'm so used to seeing this everywhere without being able to do anything that I basically ignore them now. I _think_ in some cases it's to ensure that the dials flex libraries are loaded so that the reflection_table et al can be unpickled by dxtbx, in which case it _might_ be appropriate in an example source. We should consider trying to eliminate this pattern long term, though.

Reply:
Oh, I agree, I just went with the original commenter.  I haven't investigated.

Reply:
@graeme-winter, was there a specific reason for this import?

Reply:
@ndevenish I think the recommended way to load a reflection file should now be via `flex.reflection_table.from_file()`, especially if we are moving towards message pack instead of pickle.

Reply:
most likely this is no longer the case

Reply:
:+1: